src/mono/netcore/sample/wasm/wasm.mk

```diff
@@ -16,7 +16,7 @@ build:
 	EMSDK_PATH=$(realpath $(TOP)/src/mono/wasm/emsdk) $(DOTNET) publish $(DOTNET_Q_ARGS) $(WASM_DEFAULT_BUILD_ARGS) $(MSBUILD_ARGS) $(PROJECT_NAME)
 
 clean:
-	rm -rf bin $(TOP)/artifacts/obj/mono/$(PROJECT_NAME)
+	rm -rf bin $(TOP)/artifacts/obj/mono/$(PROJECT_NAME:%.csproj=%)
```

Comment:
Out of curiosity, what does this do? What is the problem being fixed with this target?

Reply:
`PROJECT_NAME` has the filename including `.csproj`, but the directory only uses the name. So, this construct replaces the `.csproj` with empty string. `%` acting as the wildcard. Though maybe I should update the values for this var to have just the name, or change it's name.